src/packages/documents/document-types/workspace/views/design/document-type-workspace-view-edit-tab.element.ts

```diff
@@ -117,12 +117,12 @@ export class UmbDocumentTypeWorkspaceViewEditTabElement extends UmbLitElement {
 						}
 					</div>
 					<umb-document-type-workspace-view-edit-properties
-						container-id=${group.id}
+						container-id=${ifDefined(group.id)}
 						container-type="Group"
 						container-name=${group.name || ''}></umb-document-type-workspace-view-edit-properties>
 				</uui-box>`
 			)}
-			<uui-button id="add" look="placeholder" @click=${this.#onAddGroup}> Add Group </uui-button>
+			<uui-button label="Add Group" id="add" look="placeholder" @click=${this.#onAddGroup}> Add Group </uui-button>
```

Comment:
```suggestion
			<uui-button label=${this.localize.term('contentTypeEditor_addGroup')} id="add" look="placeholder" @click=${this.#onAddGroup}></uui-button>
```

Reply:
Need the translation <uui-button>here</uui-button> as well, not only the label 😊